recipes/temmeta/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "temmeta" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 97f76879bcc6b5ae9f06413dcb759e44cae824ef6384813c3bfe882449b4d35e
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas
+    - numpy
+    - matplotlib-base
+    - scipy
+    - jupyter
+    - hyperspy
+    - matplotlib-scalebar
+    - tqdm
+    - pillow
+    - py-opencv 
+    - pyqt
+
+test:
+  imports:
+    - temmeta
+
+about:
+  home: https://github.com/din14970/TEMMETA
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
Could you add a note as a comment about the (now merged) upstream issue/PR, just so there is a visual reminder for the next time the package is updated to remove the manually packaged license file

Reply:
Sure, I di it just now.